src/gameskyrim.cpp

```diff
@@ -103,7 +103,9 @@ bool GameSkyrim::isActive() const
 
 QList<PluginSetting> GameSkyrim::settings() const
 {
-  return QList<PluginSetting>();
+  QList<PluginSetting> results;
+  results.push_back(PluginSetting("sse_downloads", "allow Skyrim SE downloads", QVariant(false)));
+  return results;
```

Comment:
A matter of style maybe but this can be shortened to

```cpp
return {
    PluginSetting("sse_downloads", "allow Skyrim SE downloads", QVariant(false))
};
```